src/main/resources/static/css/custom.css

```
@@ -1206,22 +1215,460 @@ div .retxtg {padding-top:2px; font-size:13px; color:gray !important;}
+    align-items: flex-end;
+}
+
+.fl-agent-chat-message.assistant {
+    align-items: flex-start;
+}
+
+.fl-agent-chat-bubble {
+    max-width: 85%;
+    padding: 12px 16px;
+    border-radius: 12px;
+    word-wrap: break-word;
```

Stylelint (17.3.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)